MakeHistos/batch/GenerateBatchScripts.py

```diff
@@ -35,30 +35,32 @@
 ## Root macro to run from each job
 # MACRO = 'macros/ReadNTupleChain.C'
 # MACRO = 'macros/MC_data_comparison.C'
-
-#MACRO = 'macros/WH_lep.C'
 MACRO = 'macros/ttH_3l.C'
 #MACRO = 'macros/MiniNTupliser.C'
+#MACRO = 'macros/MiniNTupliser_4l_cat.C'
+#MACRO = 'macros/lepMVA_SF_calc.C'
 #MACRO = 'macros/lepMVA_efficiency.C'
 #MACRO = 'macros/lepMVA_variables.C'
 
-LOC    = 'CERN'  ## Location of input files ('CERN', 'CERN_hiM', or 'UF')
+#LOC    = 'CERN'  ## Location of input files ('CERN', 'CERN_hiM', or 'UF')
 #LOC   = 'CERN_lepMVA_test_v2'  ## Location of input files ('CERN', 'CERN_hiM', or 'UF', or 'CERN_lepMVA_test_v1')
+LOC   = 'CERN_lepMVA_3l_test_v1_with_extra_samples'
 YEAR   = 2017    ## Dataset year (2016 or 2017)
 LUMI   = 41000   ## 36814 for 2016, 41000 for 2017
 ## Override default sample location from SampleDatabase.py (use IN_DIR = '' to keep default)
+IN_DIR = ''
 IN_DIR  = '/eos/cms/store/group/phys_higgs/HiggsExo/H2Mu/UF/ntuples/2017/94X_v2/2019_01_15_LepMVA_3l_test_v1'
```

Comment:
This line overrides the line above it: should have one or the other commented-out